meson.build

```diff
@@ -67,31 +100,42 @@ add_project_arguments(
     '-Wnested-externs',
     '-Wstrict-prototypes',
   ),
-  language: 'c',
+  language : 'c',
 )
 library_options = ['-DBUILDING_LIBDICOM'] + cc.get_supported_arguments(
   '-fvisibility=hidden'
 )
 
+library_includes = include_directories('include')
```

Comment:
Nit: should these new additions also be under the `# libdicom` section?  They're sort of floating under `# compiler options` currently.

Reply:
Good idea, I'll move it.

Off-topic: how about using the `gnu_symbol_visibility` arg to library() instead of `-fvisibility`?

Reply:
Oh, do you mean the version stuff? In my mind that was building off the `version` field in the initial project() declaration, so it belonged at the top.

Reply:
Right, I think the version calculation makes sense at the top.  I meant the `library_includes` and `version.h` stuff.

Whoops, I had missed `gnu_symbol_visibility`.  Yeah, let's use that.

Reply:
OK, I added `gnu_symbol_visibility` and reorganised the sections.

How about splitting the second half of `meson.build` off into `src/meson.build` etc.? The sections that build and install files feel logically separate from the config parts to me.

Reply:
I'll make another PR, this is already getting a bit messy.